backend/src/controllers/cars/cars.controller.ts

```diff
@@ -10,12 +10,18 @@ import * as carsService from '@services/cars/cars.service';
 import { NextFunction, Request, Response } from 'express';
 
 const carsSearchAutoria = async (
-  req: TypedRequestQuery<CarsSearchParams>,
+  req: TypedRequestQuery<{
+    complectationId: string[];
+    page: string;
+  }>,
   res: Response,
   next: NextFunction,
 ): Promise<void> => {
```

Comment:
I have a few questions: 
1. `complectationId` is singular, but its type is array :eyes:?
2. Why do we allow to pass multiple complectations at once? We will show autoria options for only one complication at a time

Reply:
On the page where we can choose complectations, we use checkbox so it looks like we can specify many complactations

Reply:
Nope, the checkbox was removed from the design and it was discussed on the meetings.

Also, on e-katalog the checkbox is responsible for adding of an item to the comparison